### DISCUSSION TITLE
AWS DeepRacer Device Software OpenSource

### DISCUSSION THREAD

**Discussion by siddalingesha-ds** [aws-deepracer/aws-deepracer-follow-the-leader-sample-project#4] [General]
Whether you’d like to mount a nerf cannon on your DeepRacer with the [DeepBlaster](https://github.com/CloudBrigade/cloudbrigade-deepblaster) project, get the car to follow an object with the [Follow the Leader](https://github.com/aws-deepracer/aws-deepracer-follow-the-leader-sample-project) project, create visualizations of your home or office with the [Mapping](https://github.com/aws-deepracer/aws-deepracer-mapping-sample-project) project, or come up with new ways of racing your friends and colleagues with the [DeepDriver](https://github.com/jochem725/deepdriver) project; you can do all that and more with the AWS DeepRacer hardware, open source device software, and sample projects. 

If you create your own project, please email us at deepraceropensource@amazon.com and we’d be happy to feature it on the [Projects](https://github.com/aws-deepracer/aws-deepracer-launcher/blob/main/projects.md) list.

**Comment by lupixi** [comment]
Hi, should I must buy a deepracer kit first, or maybe could I try the project on a raspberry or jetson nano board?

**Reply by pratik-nichat** [reply]
Hello @lupixi,
Thank you for going through the repositories. At this time, we support running the sample projects on AWS DeepRacer hardware
Thank you.
